milestones/3_human_brain_mapping_journal_submission/experiments/ex_1_pli_generation_comparison/README.md

```diff
@@ -52,6 +53,17 @@ This version yield the following result:
 | B/U  | 85% | 69% |
 | B/PR | 68% | 68% |
```

Comment:
What does B,U,PR stands for?

Reply:
I found it, but it is only in a hidden part. Maybe put it somewhere visible at the beginning